lib/pbench/agent/results.py

```diff
@@ -143,7 +143,7 @@ def __init__(self, controller, tarball, config, logger):
         self.tarball_md5 = Path(f"{tarball}.md5")
         self.logger = logger
         server_rest_url = config.get("results", "server_rest_url")
-        self.upload_url = f"{server_rest_url}/upload/ctrl/{controller}"
+        self.upload_url = f"{server_rest_url}/upload/{controller}/localhost"
```

Comment:
This shouldn't work, because the Flask URI pattern matches the original version,

`f"{base_uri}/upload/ctrl/<string:controller>"`

Reply:
Done